projects/plugins/crm/.gitignore

```diff
@@ -1,37 +1,20 @@
-.DS_STORE
-.idea/
-.phpcs.xml
-*.DS_Store
-**/.DS_Store
-/.DS_STORE
-/build/
-/node_modules/
-/vendor/
-codeception.yml
-**/css/*.min.rtl.*
-**/css/**/*.min.rtl.*
-**/css/*.rtl.css
-**/css/**/*.rtl.css
-**/css/*.min.css
-**/css/welcome-to-zbs/*.min.css
+vendor/
+node_modules/
+build/
+.cache/
+**/css/*.css 
+**/css/**/*.css
 **/css/integrations/*
-**/css/font-awesome.css
+**/js/*.min.js.map
+**/js/**/*.min.js.map
 **/js/*.min.js
 **/js/welcome-to-zbs/*.min.js
 **/js/*.min.js.LICENSE.txt
-.cache/
-phpcs.xml
-phpcs.security.xml
 templates/.htaccess
 templates/emails/.htaccess
 templates/emails/index.html
 templates/index.html
 templates/invoices/.htaccess
 templates/invoices/index.html
 templates/quotes/.htaccess
-templates/quotes/index.html
-tests/acceptance.suite.yml
```

Comment:
I'm not sure we want to remove the first one; that's used for running acceptance tests with a local config.

Reply:
Ah thanks! I did not see it in my local environment so assumed it was outdated. I'll add that back.